shared/build.gradle.kts

```diff
@@ -12,5 +12,7 @@ the<DependencyManagementExtension>().apply {
 }
 
 dependencies {
+    implementation("org.testcontainers:testcontainers:1.12.3")
```

Comment:
I understand why `implementation` and not `testImplementation` ... but I'm a bit worried about it, because the dependency (and its dependencies if any) will now be part of the produced jars.

Reply:
https://docs.gradle.org/current/userguide/java_testing.html#sec:java_test_fixtures looks promising